# Room Exploration action
# sends an occupancy grid map of a room to the server, which plans a path trough it to inspect the whole room by camera

# goal definition
sensor_msgs/Image input_map				# the action server need a map as a input image to segment it, IMPORTANT: The algorithm needs a black and white 8bit single-channel image, which is 0 (black) for obstacles and 256 (white) for free space
float32 map_resolution					# the resolution of the map in [meter/cell]
geometry_msgs/Pose2D map_origin				# the origin of the map in [meter]
float32 robot_radius					# in [meter]
geometry_msgs/Point32[] field_of_view			# the points that define the field of view of the robot, relatively to the robot, [meter]
int32 path_planning_algorithm				#  this variable selects the algorithm for path planning
								# 1 = grid point explorator
geometry_msgs/Pose2D starting_position			# Pose where the robot starts in the room coordinatesystem [meter]
---
# result definition
bool inspected_room
---
# feedback definition
